content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -192,60 +201,78 @@ application in AppID and manages user authentication with builtin identity
 providers (Cloud Directory, SAML, social log-in with Google or Facebook etc.) or
 custom providers.
 
-1. Follow the guide [Getting started with App ID](https://cloud.ibm.com/docs/appid?topic=appid-getting-started)
-to create an AppID service instance.
-2. Follow the step [Registering your app](https://cloud.ibm.com/docs/appid?topic=appid-app#app-register)
+1. Setup environment variables:
+    ```
+    export KF_NAME=<your choice of name for the Kubeflow deployment>
+
+    # Set the path to the base directory where you want to store one or more 
+    # Kubeflow deployments. For example, /opt/.
+    export BASE_DIR=<path to a base directory>
+
+    # Then set the Kubeflow application directory for this deployment.
+    export KF_DIR=${BASE_DIR}/${KF_NAME}
+    ```
+2. Setup configuration files:
+    ```
```

Comment:
- Add blank lines + use "shell" formatting for code blocks

```suggestion
    ```

2. Setup configuration files:

    ```shell
```